projects/go-lib/src/lib/components/go-off-canvas/go-off-canvas.component.scss

```diff
@@ -61,3 +77,6 @@
   overflow-y: auto;
   overflow-x: hidden;
 }
+.go-off-canvas__content.large {
```

Comment:
Same here about using a modifier class, perhaps `.go-off-canvas__content--large`?

Reply:
Actually looks like this is not even being used and I can delete it.